app/support/order_uncanceler.rb

```diff
@@ -31,10 +31,10 @@ def uncancel_to_complete(order_detail)
 
         order_detail.assign_price_policy(fulfilled_at)
         order_detail.save!
-        puts "#{order_detail} was uncanceled"
+        Rails.logger.info "OrderDetail #{order_detail} was uncanceled"
         true
       rescue => e
-        puts "Could not save #{order_detail} because #{e.message}"
+        Rails.logger.info "Could not save OrderDetail #{order_detail} because #{e.message}"
         raise ActiveRecord::Rollback
       end
     end
```

Comment:
Was there a reason we were using `puts` here?  If `Rails.logger` is acceptable, that'll prevent the specs from printing a bunch of `puts` output.